src/components/vm/nics/nicBody.jsx

```diff
@@ -97,7 +97,8 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
             {
                 name: 'direct',
                 desc: 'Direct attachment',
-                detailParagraph: _("Uses the macvtap driver to attach the guest's NIC directly to a specified physical interface of the host machine.")
+                detailParagraph: _("Uses the macvtap driver to attach the guest's NIC directly to a specified physical interface of the host machine."),
+                detailHeadline: _("This config is suitable where high performance or enhanced security is required. Varying degrees of isolation between guests, or between the guest and the host are possible, depending on the mode and the capabilities of the network to which the physical interface is connected.")
```

Comment:
I've avoided using the magic jargon of VEPA/Reflective Relay/Hairpinning because cockpit-machines doesn't let you choose the `mode` of a Direct attachment network type. If it did then I think the tooltip for that option (`vepa`/`bridge`/`private`/`passthrough`) would be the appropriate place to explain the details.

Reply:
The default mode is VEPA. Do you think it needs explanation in this case?